ocaml/src/tools/etcd.ml

```diff
@@ -0,0 +1,48 @@
+(*
```

Comment:
A separate library would indeed be better, but (at least for now) I can live with it being in arakoon.
As long as we try to avoid code duplication it's fine for me